packages/builder/src/registry.ts

```diff
@@ -86,28 +91,37 @@ export class CannonRegistry {
   async readIpfs(urlOrHash: string): Promise<Buffer> {
     const hash = urlOrHash.replace(/^ipfs:\/\//, '');
 
-    const bufs: Uint8Array[] = [];
-
     debug(`downloading content from ${this.url}/${hash}`);
 
-    const response = await fetch(`${this.url}/${hash}`);
-    const blob = await response.blob();
-    const arrayBuffer = await blob.arrayBuffer();
-    return Buffer.from(arrayBuffer);
+    const chunks = [];
+    for await (const chunk of this.ipfs.cat(urlOrHash)) {
```

Comment:
I think this is revering a change made by @noahlitvin yesterday

Reply:
yep known issue